Makefile

```diff
@@ -104,6 +104,10 @@ lint: $(GOLANGCI_LINT)
 $(GOLANGCI_LINT):
 	./hack/install-golangci-lint.sh
 
+.PHONY: manifest-lint
+manifest-lint: ## Run manifest validation
```

Comment:
What do you think about adding this as a dependency of the `lint` target, too? Is that overkill?

Reply:
Yes, I think we can add it. I just added small string text so that whoever runs `make lint` can differentiate where `manifestlint.sh` finished and from where `install_golangci-lint.sh` started. Here is an example output.

```
$ make lint
./hack/install-golangci-lint.sh
+ OUTPUT=bin/golangci-lint
+ '[' -d vendor ']'
+ CMDPATH=/home/fmuyassarov/go/pkg/mod/github.com/golangci/golangci-lint@v1.32.0/cmd/golangci-lint
+ '[' '!' -f /home/fmuyassarov/go/pkg/mod/github.com/golangci/golangci-lint@v1.32.0/cmd/golangci-lint ']'
+ go mod download
+ go build -o bin/golangci-lint /home/fmuyassarov/go/pkg/mod/github.com/golangci/golangci-lint@v1.32.0/cmd/golangci-lint
./hack/manifestlint.sh
+ IS_CONTAINER=false
+ CONTAINER_RUNTIME=docker
+ [ false != false ]
+ docker run --rm --env IS_CONTAINER=TRUE --volume /home/fmuyassarov/nordix/linter/baremetal-operator:/workdir:ro,z --entrypoint sh --workdir /workdir garethr/kubeval:latest /workdir/hack/manifestlint.sh
+ IS_CONTAINER=TRUE
+ CONTAINER_RUNTIME=podman
+ '[' TRUE '!=' false ]
<-------------------------STARTING MANIFESTS VALIDATION CHECKS------------------------->
WARN - Set to ignore missing schemas
1..59
ok 1 #skip - config/certmanager/certificate.yaml (Issuer)
ok 2 #skip - config/certmanager/certificate.yaml (Certificate)
ok 3 #skip - config/crd/bases/metal3.io_baremetalhosts.yaml
ok 4 #skip - config/crd/bases/metal3.io_baremetalhosts.yaml (CustomResourceDefinition)
ok 5 #skip - config/crd/patches/cainjection_in_baremetalhosts.yaml (CustomResourceDefinition)
ok 6 #skip - config/crd/patches/webhook_in_baremetalhosts.yaml (CustomResourceDefinition)
ok 7 - config/manager/manager.yaml (Deployment)
ok 8 - config/namespace/namespace.yaml (Namespace)
ok 9 #skip - config/prometheus/monitor.yaml (ServiceMonitor)
ok 10 - config/rbac/auth_proxy_client_clusterrole.yaml (ClusterRole)
ok 11 - config/rbac/auth_proxy_role.yaml (ClusterRole)
ok 12 - config/rbac/auth_proxy_role_binding.yaml (ClusterRoleBinding)
ok 13 - config/rbac/auth_proxy_service.yaml (Service)
ok 14 - config/rbac/baremetalhost_editor_role.yaml (ClusterRole)
ok 15 - config/rbac/baremetalhost_viewer_role.yaml (ClusterRole)
ok 16 - config/rbac/leader_election_role.yaml (Role)
ok 17 - config/rbac/leader_election_role_binding.yaml (RoleBinding)
ok 18 #skip - config/rbac/role.yaml
ok 19 - config/rbac/role.yaml (ClusterRole)
ok 20 - config/rbac/role_binding.yaml (ClusterRoleBinding)
ok 21 #skip - config/render/capm3.yaml (CustomResourceDefinition)
ok 22 - config/render/capm3.yaml (Role)
ok 23 - config/render/capm3.yaml (ClusterRole)
ok 24 - config/render/capm3.yaml (ClusterRole)
ok 25 - config/render/capm3.yaml (ClusterRole)
ok 26 - config/render/capm3.yaml (RoleBinding)
ok 27 - config/render/capm3.yaml (ClusterRoleBinding)
ok 28 - config/render/capm3.yaml (ClusterRoleBinding)
ok 29 - config/render/capm3.yaml (ConfigMap)
ok 30 - config/render/capm3.yaml (Service)
ok 31 - config/render/capm3.yaml (Deployment)
ok 32 #skip - config/samples/metal3.io_v1alpha1_baremetalhost.yaml (BareMetalHost)
ok 33 #skip - examples/demo-hosts.yaml
ok 34 - examples/demo-hosts.yaml (Secret)
ok 35 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 36 #skip - examples/demo-hosts.yaml (Machine)
ok 37 - examples/demo-hosts.yaml (Secret)
ok 38 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 39 - examples/demo-hosts.yaml (Secret)
ok 40 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 41 - examples/demo-hosts.yaml (Secret)
ok 42 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 43 - examples/demo-hosts.yaml (Secret)
ok 44 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 45 - examples/demo-hosts.yaml (Secret)
ok 46 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 47 - examples/demo-hosts.yaml (Secret)
ok 48 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 49 - examples/demo-hosts.yaml (Secret)
ok 50 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 51 - examples/demo-hosts.yaml (Secret)
ok 52 #skip - examples/demo-hosts.yaml (BareMetalHost)
ok 53 - examples/example-host-bad-credentials.yaml (Secret)
ok 54 - examples/example-host-bad-credentials.yaml (Secret)
ok 55 #skip - examples/example-host-bad-credentials.yaml (BareMetalHost)
ok 56 - examples/example-host.yaml (Secret)
ok 57 #skip - examples/example-host.yaml (BareMetalHost)
ok 58 - examples/worker-0.yaml (Secret)
ok 59 #skip - examples/worker-0.yaml (BareMetalHost)
<-------------------------COMPLETED MANIFESTS VALIDATION CHECKS------------------------>
bin/golangci-lint run

```

Reply:
However, if we add it to the `lint`, then the user has to make sure to export `CONTAINER_RUNTIME` variable before running `make lint`.

Reply:
Actually, in the [doc](https://github.com/metal3-io/baremetal-operator/blob/6c0d397802e9ba635283484069cb81390febc716/docs/testing.md#example-of-running-golint-in-a-docker-container) we already ask first to export `CONTAINER_RUNTIME` var whenever we want to run scripts under /hack. So, I would assume it is okay to add manifestlint as a dependency on `lint` and update the document. WDYT @dhellmann ?